docs/en/sql-reference/functions/null-functions.md

```diff
@@ -52,6 +54,45 @@ Result:
 └───┘
 ```
 
+## isNullable
+
+Returns `1` if a column allows null values, `0` otherwise.
+
+**Syntax**
+
+``` sql
+isNullable(x)
+```
+
+**Arguments**
+
+- `x` — column.
+
+**Returned value**
+
+- `1` if `x` allows `NULL` values. [UInt8](../data-types/int-uint.md).
+- `0` if `x` does not allow `NULL` values. [UInt8](../data-types/int-uint.md).
+
+**Example**
+
+Query:
+
+``` sql
+CREATE TABLE example_table (nullable_column Nullable(UInt32), ordinary_column UInt32) ENGINE = Log;
```

Comment:
`example_table` --> `tab`

Reply:
Let's list the "ordinary column" first, then the "nullable_column". Also let's name them `ordinarly_col` and `nullable_col`.